README.md

```diff
@@ -18,13 +26,30 @@ Runs looking glass using the specified config and modules path.
 glass run -c /path/to/config.yaml -m /path/to/modules
 ```
 
-| Command               | Env        | Description                                                                  |
-|-----------------------|------------|------------------------------------------------------------------------------|
-|`--secrets`/`-s` FILE  | SECRETS    | The path to the secrets file.                                                |
-| `--config`/`-c` FILE  | CONFIG     | The path to the configuration file.                                          |
-| `--modules`/`-m` PATH | MODULES    | The path to the modules. Modules will be located under `src` in this folder. |
-| `--log.format` FORMAT | LOG_FORMAT | Specify the format of logs. Supported formats: 'logfmt', 'json'              |
-| `--log.level` LEVEL   | LOG_LEVEL  | Specify the log level. E.g. 'debug', 'warning'. (default: "info")            |
+#### Options
```

Comment:
WDYT of making options an h3 instead of h4 and adding it to the ToC?

Reply:
it is `options` for the `Run`. So would need to `Run Options` then?